src/backend/utils/misc/guc.c

```diff
@@ -8200,7 +8200,7 @@ static struct config_string ConfigureNamesString[] =
 			NULL
 		},
 		&standby_addr_host,
-		"localhost", NULL, NULL
+		"none", NULL, NULL
```

Comment:
If we support "" instead of "none", we have segment keep connecting to "none" as a server. I think if necessary, we need another fix for this.